src/datadoc/frontend/fields/display_base.py

```diff
@@ -163,6 +163,7 @@ def render(
 class MetadataDropdownField(DisplayMetadata):
     """Control how a Dropdown should be displayed."""
 
+    show_description: bool = True
```

Comment:
I think this should go into the top level class now since it applies to all fields, right?